README.md

```diff
@@ -1,6 +1,6 @@
 # Project specific interfaces to REMIND / MAgPIE
 
-R package **piamInterfaces**, version **0.0.114**
+R package **piamInterfaces**, version **0.1.0**
```

Comment:
Now that comments are supported, I think the first minor release is indeed justified :)

Reply:
`updateType = as.integer(Sys.Date()) %% 5`, approved by @fbenke-pik!